### PR TITLE
FIX: Touch control scheme associations in default actions.

### PR DESCRIPTION
**Pull request by Rene-Damm**
Continuing to tweak the default actions. Some touch bindings were not associated with the touch control scheme. Started adding an XR scheme but there's some controls that with the currently available built-in tools can't be satisfied properly.

### CODE REVIEW

File: Packages/com.unity.inputsystem/InputSystem/Plugins/XInput/XInputController.cs [hunk 1]
```diff
@@ -65,8 +65,8 @@ public class XInputController : Gamepad
         [InputControl(name = "leftTrigger", shortDisplayName = "LT")]
         [InputControl(name = "rightTrigger", shortDisplayName = "RT")]
         // This follows Xbox One conventions; on Xbox 360, this is start=start and select=back.
-        [InputControl(name = "start", displayName = "Menu")]
-        [InputControl(name = "select", displayName = "View")]
+        [InputControl(name = "start", displayName = "Menu", alias = "menu")]
+        [InputControl(name = "select", displayName = "View", alias = "view")]
```
Comment (Rene-Damm) [inline Packages/com.unity.inputsystem/InputSystem/Plugins/XInput/XInputController.cs:69]: Minor, unrelated fix. There's an open bug about Xbox One controls not resolving when using the "menu" and "view" names. These are already exposed as getters under those names but the controls were missing aliases to also allow lookup of control by those names.

**Review by douglas-piconi — APPROVED** [review]
